src/storage/src/error.rs

```diff
@@ -210,6 +209,18 @@ pub enum ReadError {
     #[cfg(google_cloud_unstable_storage_bidi)]
     #[error("unknown range id in bidi response: {0}")]
     UnknownBidiRangeId(i64),
+
+    /// A `read_range()` request failed because the object descriptor is closed.
+    ///
+    /// # Troubleshooting
+    ///
+    /// The object descriptor closes only when there is an unrecoverable I/O
+    /// error. Consider using a more lenient [ReadResumePolicy].
+    ///
+    /// [ReadResumePolicy]: [crate::read_resume_policy::ReadResumePolicy]
```

Comment:
optional nit: I don't think the brackets around `crate::read_resume_policy::ReadResumePolicy` are necessary, but I think it works either way